CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Added
+- Added `nix::sys::socket::ControlMessage::ScmTimestamp`
```

Comment:
I don't really understand this subsystem as it's missing pretty much all relevant documentation, but you added a new `ControlMessage` variant to handle `SCM_TIMESTAMP` messages. You also added `ReceiveTimestamp` as something, that I think is user-facing, but I don't understand what it is and how it's used, but it should probably be documented under "Added" as well if it's a public API addition.